src/components/FeedbackCreateForm/FeedbackCreateForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Button, Form, FormActions, FormAction, FormTextarea, FormInput, ModalContent } from '@taskany/bricks';
+import * as Sentry from '@sentry/nextjs';
+
+import { errorsProvider } from '../../utils/forms';
+import { createFeedbackSchema, CreateFeedback } from '../../schema/feedback';
+import { ModalEvent, dispatchModalEvent } from '../../utils/dispatchModal';
+import { dispatchErrorNotification, dispatchSuccessNotification } from '../../utils/dispatchNotification';
+
+import { tr } from './FeedbackCreateForm.i18n';
+
+const FeedbackCreateForm: React.FC = () => {
+    const [formBusy, setFormBusy] = useState(false);
+
+    const {
+        register,
+        handleSubmit,
+        formState: { errors, isSubmitted },
+    } = useForm<CreateFeedback>({
+        resolver: zodResolver(createFeedbackSchema),
+    });
+
+    const errorsResolver = errorsProvider(errors, isSubmitted);
+
+    const onPending = useCallback(async (form: CreateFeedback) => {
+        setFormBusy(true);
+        try {
+            const res = await fetch('/api/feedback', {
+                method: 'POST',
+                headers: {
+                    'content-type': 'application/json',
+                    accept: 'application/json',
+                },
+                body: JSON.stringify({
+                    title: form.title,
+                    description: form.description,
+                    href: window.location.href,
+                }),
+            });
+            if (res.status < 400) {
+                dispatchModalEvent(ModalEvent.FeedbackCreateModal)();
+                dispatchSuccessNotification('sentFeedback');
```

Comment:
shouldn't it be a success text from locale?

Reply:
elsewhere in the project code dispatchSuccessNotificatio is used without 'tr'
we have NotificationHub.map - https://github.com/taskany-inc/issues/blob/main/src/components/NotificationsHub/NotificationHub.map.ts